add profiler doc (#1119)
PaddlePaddle/VisualDL: README.md

<img src="https://user-images.githubusercontent.com/28444161/119247155-e9c0c280-bbb9-11eb-8175-58a9c7657a9c.gif" width="85%"/>
</p>

### Performance Analysis

> **Performance Analysis**(Profiler) visualize the profiling data collected during your program runs，and helps you recognize the bottleneck of your program and optimize performance. Please refer to [VisualDL Profiler Guide](./profiler/README.md)。

Reviewer: ```suggestion
**Performance Analysis**(Profiler) visualize profiling data collected during program runs，helping you identify program bottlenecks and optimize performance. Please refer to [VisualDL Profiler Guide](./profiler/README.md)。
```

Author: Done
